README.md

```diff
@@ -31,249 +31,23 @@ boilerplate code as new improvements are discovered.
 Is actively being worked on and supported by TrueCar.
 
 ## Requirements
-GlueStick works best with v6.x.x of node. Versions prior to v6 are not currently supported and node v7 has known issues: https://github.com/TrueCar/gluestick/issues/322
+GlueStick works best with v6.x.x of node. Versions prior to v6 are not currently supported.
```

Comment:
Versions prior to 6.3.0 are not supported, and it works as good with node 6.3.0+ as with node 7